instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -67,9 +88,12 @@
     _SUPPRESS_INSTRUMENTATION_KEY,
     http_status_to_status_code,
 )
+from opentelemetry.metrics import get_meter
+from opentelemetry.metrics._internal.instrument import Histogram
```

Comment:
Why `_internal`? Is it not part of public api?

Reply:
Changed to import from `opentelemetry.metrics`.